types/src/gens.rs

```diff
@@ -626,6 +626,7 @@ pub fn package_arb() -> impl Strategy<Value = Package> {
     )
 }
 
+#[allow(unused)]
```

Comment:
Intentional?

Reply:
It is now unused, in favor of delegator_bid_arb()